ui-v2/src/components/concurrency/task-run-concurrenct-view/dialogs/delete-dialog.tsx

```diff
@@ -0,0 +1,69 @@
+import { Button } from "@/components/ui/button";
+import {
+	Dialog,
+	DialogContent,
+	DialogDescription,
+	DialogFooter,
+	DialogHeader,
+	DialogTitle,
+	DialogTrigger,
+} from "@/components/ui/dialog";
+import {
+	TaskRunConcurrencyLimit,
+	useDeleteTaskRunConcurrencyLimit,
+} from "@/hooks/task-run-concurrency-limits";
+import { useToast } from "@/hooks/use-toast";
+
+type Props = {
+	data: TaskRunConcurrencyLimit;
+	onOpenChange: (open: boolean) => void;
+	onDelete: () => void;
+};
+
+export const DeleteLimitDialog = ({ data, onOpenChange, onDelete }: Props) => {
+	const { toast } = useToast();
+	const { deleteTaskRunConcurrencyLimit, isPending } =
+		useDeleteTaskRunConcurrencyLimit();
+
+	const handleOnClick = (id: string | undefined) => {
+		if (!id) {
+			throw new Error("'id' field expected in GlobalConcurrencyLimit");
```

Comment:
nit, but we won't need this much longer.
```suggestion
			throw new Error("'id' field expected in TaskRunConcurrencyLimit");
```